library/components/TPdf.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-    <div class="relative">
+    <div class="relative" @mouseover="showTooltip = true" @mouseleave="showTooltip = false">
```

Comment:
Events have strange behaviour when used on button, using in container div works fine.

Reply:
interesting, like what?

Reply:
For me, it was not even triggering these events for some reason, even though it should tbh, not sure what caused it, not a huge concern in this case, but this is something I would like to check deeply, what did I miss.

Reply:
I tested putting the mouseover/leave on the button and it  seemed to work 🤷 

I noticed though that the tooltip is not very easy to keep open because it is placed a bit away from the button, it is hard to hover over

Reply:
In this case maybe it's not going to matter to keep it open, but I see what you mean, in some cases there might be a link to click inside, this is something to fix inside TTooltip component, I'll take a look, thanks.